syft_tensorflow/hook/hook.py

```diff
@@ -284,7 +284,7 @@ def _add_methods_from_native_tensor(tensor_type: type, syft_type: type):
 
     @classmethod
     def create_wrapper(cls, child_to_wrap, *args, **kwargs):
-        if isinstance(child_to_wrap.object_type, tf.Variable):
+        if type(child_to_wrap.object_type)==type(tf.Variable):
```

Comment:
why the change? type(x) == type(y) is discouraged by PEP

Reply:
see replied to Ben